connectors/analytics/conviva/src/main/java/com/theoplayer/android/connector/analytics/conviva/utils/Utils.kt

```diff
@@ -138,7 +138,7 @@ fun updateAdMetadataForGoogleIma(ad: GoogleImaAd, metadata: ConvivaMetadata): Co
 fun collectAdMetadata(ad: Ad): ConvivaMetadata {
     // AssetName should never be an empty string
     return mutableMapOf(
-        ConvivaSdkConstants.DURATION to if (ad is LinearAd) ad.duration else 0,
+        ConvivaSdkConstants.DURATION to if (ad is LinearAd) ad.durationAsDouble else 0.0,
```

Comment:
A little bit more idiomatic perhaps:
```suggestion
        ConvivaSdkConstants.DURATION to (ad as? LinearAd).durationAsDouble ?? 0.0,
```